astropy/time/core.py

```diff
@@ -124,6 +124,16 @@ class Time(object):
     formats (e.g. JD) where very high precision (better than 64-bit precision)
     is required.
 
+    The allowed values for ``format`` and ``scale`` can be listed with::
+
+      >>> list(Time.FORMATS)
+      ['jd', 'mjd', 'decimalyear', 'unix', 'cxcsec', 'gps', 'plot_date', 'astropy_time',
+       'datetime', 'iso', 'isot', 'yday', 'fits', 'byear', 'jyear', 'byear_str',
+       'jyear_str']
+
+      >>> list(Time.SCALES)
+      ['tai', 'tcb', 'tcg', 'tdb', 'tt', 'ut1', 'utc']
```

Comment:
`Time.SCALES` is a `dict`, so the order is not guaranteed (which is why travis failed for python3).

How about not as much showing how to get them, but rather just inserting the values? Either just do it or make it auto-fill, using something like:
```
class Time(object):
    """
    ...
    format : str, optional
        Format of input value(s); should be one of the following:
        {0}
    scale : str, optional
        Time scale of input value(s); should be one of the following:
        {1}
    ...
    """
# and then at the very end:
Time.__doc__ = Time.__doc__.format(list(Time.FORMATS), list(Time.SCALES))
TimeDelta.__doc__ = TimeDelta.__doc__.format(list(TimeDelta.FORMATS), list(TimeDelta.SCALES))
```
(though if we reorder -- or move all the formats to `astropy.time.formats` -- the postprocessing may not be needed.)